app/Decsys/Constants/Collections.cs

```diff
@@ -34,5 +34,14 @@ public static class Collections
         public const string RandList = "RandList";
 
         #endregion
+
+        #region Wordlist DBs
+
+        public const string UserWordlist = "UserWordlist";
+        public const string WordlistOptions = "WordlistOptions";
+        public const string WordlistRules = "WordlistRules";
+        public const string WordlistWord = "WordlistWord";
```

Comment:
These don't need their own collections, since we're NoSql.

Basically these entities are nested inside a Wordlist, so the collection level is just Wordlists